dali/test/argument_key.h

```diff
@@ -15,27 +15,50 @@
 #ifndef DALI_TEST_ARGUMENT_KEY_H_
 #define DALI_TEST_ARGUMENT_KEY_H_
 
-#include <utility>
+#include <assert.h>
```

Comment:
Pedantic ;) `#include <cassert>` Fix only if you change anything else along the way, otherwise don't bother.